apps/editor/widget/app.jsx

```diff
@@ -139,6 +161,15 @@ const types = [
   },
 ];
 
+const data = Social.get("rambo-dev.near/every/document/*");
+
+const postsArray = Object.values(data).map(JSON.parse);
```

Comment:
Imagine 100+ documents, this might take a while, huh?

We know the id of the document we are editing

Reply:
I will review it, thanks!